src/Extensions/Compatability/TimedScopes/TimedScopeDefinitionExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Omex.Extensions.TimedScopes;
+
+namespace Microsoft.Omex.Extensions.Compatability.TimedScopes
+{
+	/// <summary>
+	/// Extension for TimedScopeDefinition
+	/// </summary>
+	public static class TimedScopeDefinitionExtensions
+	{
+		/// <summary>
+		/// Creates a scope (and starts by default)
+		/// </summary>
+		/// <param name="timedScopeDefinition">TimedScopeDefinition to use</param>
+		/// <param name="initialResult">Initial result to use</param>
+		/// <param name="startScope">Should the scope be automatically started (for use in e.g. 'using' statement)</param>
+		[Obsolete(ObsoleteMessage, IsObsoleteError)]
+		public static TimedScope Create(this TimedScopeDefinition timedScopeDefinition, TimedScopeResult initialResult, bool startScope = true)
```

Comment:
Missing `<returns>`

Reply:
I don't want to invest in improving comments for obsolete methods, sorry